_pages/how-we-work/sensitive-information.md

```diff
@@ -8,7 +8,7 @@ Here's what you need to know about sensitive information at TTS.
 
 ## What is considered sensitive?
```

Comment:
Is there anything unique to sensitive information within TTS as opposed to GSA? Spending time reviewing this, I'm not really seeing the ROI on having it rather than the annual training folks get on this on their own.

Reply:
Outside the scope of this pull request, but I'd say the range of what's considered sensitive within the agency varies widely, yes. Let's address separately.

Reply:
https://github.com/18F/tts-tech-portfolio/issues/928